### PR TITLE
feat(ET styles): Add icons, buttons, and some variables.

### PR DESCRIPTION
**Pull request by Camwyn**
Also added a "link-style" button mixin.

Add `close`, `error`, and `remove` icons.

**HOLD** for  merge of AR Modal.

### CODE REVIEW

File: variables/_svgs.pcss [hunk 1]
```diff
@@ -106,6 +106,25 @@
...
+ * ----------------------------------------------------------------------------- */
+
+@svg-load close url('../icons/close.svg') {
+
+	path {
+		stroke: var(--color-icon-secondary);
+	}
+}
+
+@svg-load close-focus url('../icons/close.svg') {
```
Comment (paulmskim) [inline variables/_svgs.pcss]: let's call this `close` and above `close-secondary`, to follow the icon color we're using (see other uses in this file).

**Comment by sc0ttkclark** [comment]
Is this to be code reviewed/merged or is this on hold?

**Comment by Camwyn** [comment]
> Is this to be code reviewed/merged or is this on hold?

@sc0ttkclark I was holding for it to be merged until we deploy ar-modal

File: icons/arrow.svg [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="11px" height="10px" viewBox="0 0 11 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (paulmskim) [inline icons/arrow.svg]: remove xml and minify svg

File: variables/_colors.pcss [hunk 3]
```diff
@@ -11,11 +11,12 @@
 	 * Colors - Iconography
 	 * ----------------------------------------------------------------------------- */
 
-	--color-icon-primary: #727272;
+	--color-icon-primary:   #727272;
```
Comment (paulmskim) [inline variables/_colors.pcss]: we do not align values with each other in css.

File: variables/_colors.pcss [hunk 4]
```diff
@@ -40,15 +41,16 @@
 	 * Colors - Borders & Misc
 	 * ----------------------------------------------------------------------------- */
 
-	--color-border-default: #D5D5D5;
-	--color-border-secondary: #E4E4E4;
-	--color-border-hover: #727272;
-	--color-border-active: #141827;
-	--color-background: #FFFFFF;
+	--color-border-default:         #D5D5D5;
```
Comment (paulmskim) [inline variables/_colors.pcss]: same note here on aligning values in css

File: variables/_svgs.pcss [hunk 5]
```diff
@@ -23,6 +23,17 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Arrow Left
+ * ----------------------------------------------------------------------------- */
+
+@svg-load arrow-left url('../icons/arrow.svg') {
```
Comment (paulmskim) [inline variables/_svgs.pcss]: is this arrow the same as arrow right but pointing left? if so, can we name this file so it matches the same format (`arrow-left.svg`)?

File: variables/_svgs.pcss [hunk 6]
```diff
@@ -120,6 +131,24 @@
...
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Close
+ * ----------------------------------------------------------------------------- */
+
+@svg-load close url('../icons/close.svg') {
+
+	path {
+		stroke: var(--color-icon-secondary);
```
Comment (paulmskim) [inline variables/_svgs.pcss]: `close` should use primary color and `close-secondary` should use secondary color, keeping naming consistent with color we use.

File: variables/_svgs.pcss [hunk 7]
```diff
@@ -142,6 +171,17 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Error
+ * ----------------------------------------------------------------------------- */
+
+@svg-load error url('../icons/error.svg') {
```
Comment (paulmskim) [inline variables/_svgs.pcss]: call this `error-active` if we are using active color

File: variables/_colors.pcss [hunk 8]
```diff
@@ -48,6 +49,7 @@
 	--color-background-transparent: rgba(var(--color-background), 0.6);
 	--color-background-secondary: #F7F6F6;
 	--color-background-secondary-hover: #F0EEEE;
+	--color-background-error: rgba(#DA394D, 0.08);
```
Comment (borkweb) [inline variables/_colors.pcss]:
```suggestion
	--color-background-error: rgba(var(--color-icon-error), 0.08);
```

**Comment by Camwyn** [comment]
Worth noting here that I had to preserve most of the arrow.svg or it got "janky" (to quote design) - blurry. So don't throw it through SVGOMG or something because it will become blurry again.

**Comment by paulmskim** [comment]
@Camwyn can you point this to a release branch?

**Pull request merged**